### PR TITLE
[INLONG-8449][Manager] Fix abnormal growth of self increasing primary key in heartbeat table

### LINKED ISSUE #8449: [Bug][Manager] Abnormal growth of self increasing primary key in heartbeat table
### What happened

Abnormal growth of self increasing primary key in heartbeat table.

There are only two heartbeat messages in the database, but the self increasing primary key is very high.

企业微信截图_58c84358-4469-47cd-b2ac-bc9e08186445

### What you expected to happen

Abnormal growth of self increasing primary key in heartbeat table.

### How to reproduce

Abnormal growth of self increasing primary key in heartbeat table.

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #8449 

### Motivation

 Fix abnormal growth of self increasing primary key in heartbeat table.

### Modifications

 Fix abnormal growth of self increasing primary key in heartbeat table.

### CODE REVIEW

**Review by vernedeng — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

File: inlong-manager/manager-dao/src/main/resources/mappers/ComponentHeartbeatEntityMapper.xml [hunk 1]
```diff
@@ -49,17 +48,11 @@
         values (#{component,jdbcType=VARCHAR}, #{instance,jdbcType=VARCHAR},
                 #{statusHeartbeat,jdbcType=LONGVARCHAR}, #{metricHeartbeat,jdbcType=LONGVARCHAR},
                 #{reportTime,jdbcType=BIGINT})
-        ON DUPLICATE KEY UPDATE status_heartbeat = values(status_heartbeat),
-                                metric_heartbeat = values(metric_heartbeat),
-                                report_time      = values(report_time)
+            ON DUPLICATE KEY UPDATE status_heartbeat = values(status_heartbeat),
+            metric_heartbeat = values(metric_heartbeat),
+                                 report_time      = values(report_time)
```
Comment (healchow) [inline inlong-manager/manager-dao/src/main/resources/mappers/ComponentHeartbeatEntityMapper.xml:53]: Please format those lines.

File: inlong-manager/manager-web/sql/changes-1.8.0.sql [hunk 2]
```diff
@@ -118,3 +118,15 @@ ALTER TABLE `workflow_process`
...
+ALTER TABLE `component_heartbeat` ADD PRIMARY KEY (`component`, `instance`);
+DROP INDEX unique_component_heartbeat on component_heartbeat;
+
+ALTER TABLE group_heartbeat DROP COLUMN id;
+ALTER TABLE `group_heartbeat` ADD PRIMARY KEY  (`component`, `instance`, `inlong_group_id`);
+DROP INDEX unique_group_heartbeat on group_heartbeat;
+
+ALTER TABLE stream_heartbeat DROP COLUMN id;
+ALTER TABLE `stream_heartbeat` ADD PRIMARY KEY (`component`, `instance`, `inlong_group_id`, `inlong_stream_id`);
+DROP INDEX unique_stream_heartbeat on stream_heartbeat;
```
Comment (healchow) [inline inlong-manager/manager-web/sql/changes-1.8.0.sql:132]: Please add one blank line at the end of file.

Comment (fuweng11) [reply inlong-manager/manager-web/sql/changes-1.8.0.sql:136]: Done

File: inlong-manager/manager-dao/src/main/resources/mappers/ComponentHeartbeatEntityMapper.xml (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-dao/src/main/resources/mappers/ComponentHeartbeatEntityMapper.xml]: Done

**Review by healchow — APPROVED** [review]

**Pull request merged**